source/loader/ur_ldrddi.cpp

```diff
@@ -5295,5 +5296,5 @@ urGetDeviceProcAddrTable(
 }
 
 #if defined(__cplusplus)
-};
+}
```

Comment:
I think this was covered in this [PR](https://github.com/oneapi-src/unified-runtime/pull/278/files)

Reply:
I've rebased now :)